capstone/src/main/webapp/profile.html

```diff
@@ -76,7 +78,7 @@ <h2 id="modal-name-container"></h2>
             <input type="text" class="input-field" placeholder="Add an interest" 
               name="interests" id="interests">
           </li>          
-          <button class="save-btn" onclick="saveEdits()">SAVE</button>
+          <button class="save-btn">SAVE</button>
```

Comment:
Same as above. I tested this out and it works, but I'm sort of confused why? I'm not super familiar with html forms but I thought the save button would need to have a type="submit"

Reply:
Blobstore handles forms differently. Explained above.